spec/jobs/state_file/send_reject_resolution_reminder_notification_job_spec.rb

```diff
@@ -57,6 +57,37 @@
             message: message,
             body_args: body_args).exactly(2).times
         end
+
+        context "with unverified phone number" do
+          before do
+            intake.update(phone_number: "+13453432222", phone_number_verified_at: nil, sms_notification_opt_in: "yes", email_address: 'test@example.com', email_address_verified_at: 5.minutes.ago, email_notification_opt_in: "no")
+          end
+
+          it "should still send the message" do
+            expect {
+              described_class.perform_now(intake)
+            }.to change(StateFileNotificationTextMessage, :count).by(1)
+
+            expect(intake.reload.message_tracker).to include("messages.state_file.reject_resolution_reminder")
+
+            expect(StateFile::MessagingService).to have_received(:new).with(
+              intake: intake,
+              message: message,
+              body_args: body_args)
+
+            # can re-send if message was sent before (send 13th & 23rd in 2025)
+            expect {
+              described_class.perform_now(intake)
+            }.to change(StateFileNotificationTextMessage, :count).by(1)
+
+            expect(intake.reload.message_tracker).to include("messages.state_file.reject_resolution_reminder")
+
+            expect(StateFile::MessagingService).to have_received(:new).with(
+              intake: intake,
+              message: message,
+              body_args: body_args).exactly(2).times
+          end
+        end
```

Comment:
stuck into one of the happy-path tests, but happy to duplicate it in the submssion that is currently in `notified_of_rejection`